woocommerce/woocommerce
# NOX: Manage Active Gateways

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #51446 

* Hooks NOX Payment Gateways section up to fetch gateways from the backend.
* Implement logic to show correct buttons for each gateway.
* Some (not all) functionality of buttons implemented

This PR does **not** include:
* Gateway recommendations on a region-specific basis
* Offline payment methods or gateway recommendations
* Implement drag-and-drop sorting of gateways (will pick this up as a separate issue)

Known Issues
* Currently we don't have icons being returned on a per gateway basis, we'll need to figure out a solution to match the designs.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

See the instructions for this PR: https://github.com/woocommerce/woocommerce/pull/52030



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 

</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
